io.github.maurycyliebner.enve.yaml

```diff
@@ -0,0 +1,186 @@
+app-id: io.github.maurycyliebner.enve
+default-branch: stable
```

Comment:
```suggestion
```

Reply:
I put it in there, to make it reusable for possible beta branch, but ok, I will just remove it.